src/Misc/layoutbin/powershell/Add-AwsCliCapabilities.ps1

```diff
@@ -0,0 +1,4 @@
+[CmdletBinding()]
+param()
+
+Add-CapabilityFromApplication -Name 'AWS_Cli' -ApplicationName 'aws'
```

Comment:
Not sure where I should start with this.  Can you point me in the right direction?